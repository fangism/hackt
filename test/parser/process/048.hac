// "048.hac" should fail: ports mismatch

defproc food(int<16> sleep);

defproc food(int sleep);		// default 32

