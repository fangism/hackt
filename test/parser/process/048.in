// "048.in" should fail: ports mismatch

defproc food(int<16> sleep);

defproc food(int sleep);		// default 32

