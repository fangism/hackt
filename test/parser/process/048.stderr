ERROR: port formals do not match!
new declaration for "food" doesn't match previous declaration.  ERROR! [5:1..23]
